index.html

```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio</title>
+    <link rel="stylesheet" type="text/css" href="style.css">
+</head>
+<body>
+      <div class="nav">
```

Comment:
- [ ] The container of the section should be `<header>` tag. Change div into header and problem will be solved 👨‍🔧. Using semantic html elements makes your code better like: `<article>`, `<aside>`, `<time>` etc. 😃👌